universal/ubi8/Dockerfile

```diff
@@ -16,7 +16,7 @@ LABEL description="Image with developers tools. Languages SDK and runtimes inclu
 LABEL io.k8s.display-name="devfile-developer-universal"
 LABEL io.openshift.expose-services=""
 
-USER 1001
+USER 10001
```

Comment:
do we really need these changes?

Reply:
I suppose `1001` here is a typo

Reply:
I wonder if the real user ID in a container is 1234, why do we use 10001 instead?
```
bash-4.4$ id     
uid=1234(user) gid=0(root) groups=0(root)
```

Reply:
The user ID is set by the underlying container runtime. It can be a random one (`1234`), the default one (`10001`) or a specific one (`0`). `10001` is a user that is not privileged.

And yes, that's probably a typo.